storage/innobase/dict/dict0stats.cc

```diff
@@ -138,6 +138,13 @@ data:  b,b,b,b,b,b,g,g,j,j,j, x, y
 then we would store 5,7,10,11,12 in the array. */
 typedef std::vector<ib_uint64_t, ut_allocator<ib_uint64_t> >	boundaries_t;
 
+/** Structure to hold leaf page statistics */
+struct LeafPageStats {
+	ib_uint64_t n_diff_on_leaf_page = 0;
+	ib_uint64_t n_external_pages = 0;
+	ib_uint64_t n_non_null_on_leaf_page = 0;
+};
```

Comment:
The `_on_leaf_page` and `_pages` suffixes are redundant. Can we use the standard `uint64_t` type in new code?

Also, I’d remove the initializer expressions. If this needs to be zero-initialized, it’d be clearer to do it explicitly rather than in an implicly created constructor.